multimodal-understanding/repeatable-patterns/10-multimodal-rag-with-langchain/multimodal_rag_with_nova.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    with tqdm(total=len(items), desc="Generating embeddings", bar_format="{l_bar}{bar}| {n_fmt}/{total_fmt} [{elapsed}&lt;{remaining}, {rate_fmt}{postfix}]") as pbar:
</pre><p>nit: consider to use a formatter "black[jupyter]". Some lines are a little long to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-nova-samples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>